app/controllers/home_controller.rb

```diff
@@ -131,7 +131,7 @@ def get_id(path, user, classname)
       when "nomination"
         TagSetNomination.for_tag_set(OwnedTagSet.find(@last_id)).owned_by(user).first
       when "setting"
-        AdminSetting.first
+        Rails.cache.fetch("admin_settings"){AdminSetting.first}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.